java/spacewalk-java.spec

```diff
@@ -32,34 +32,27 @@
 %define appdir          /srv/tomcat/webapps
 %define jardir          /srv/tomcat/webapps/rhn/WEB-INF/lib
 %define apache_group    www
+%define salt_user-group    salt
 %define apache2         apache2
-%define ehcache         ehcache >= 2.10.1
-%define apache_commons_discovery   jakarta-commons-discovery
-%define apache_commons_el          jakarta-commons-el
-%define apache_commons_validator   jakarta-commons-validator
 %define java_version    11
-%if 0%{?sle_version} > 150100
-%define apache_commons_fileupload  apache-commons-fileupload
-%define apache_commons_digester    apache-commons-digester
-%define log4j                      log4j12
-%else
-%define apache_commons_fileupload  jakarta-commons-fileupload
-%define apache_commons_digester    jakarta-commons-digester
-%define log4j                      log4j
-%endif
 %else
 %define appdir          %{_localstatedir}/lib/tomcat/webapps
 %define jardir          %{_localstatedir}/lib/tomcat/webapps/rhn/WEB-INF/lib
 %define apache_group    apache
+%define salt_user_group root 
 %define apache2         httpd
-%define ehcache         ehcache-core >= 2.10.1
-%define apache_commons_discovery   apache-commons-discovery
-%define apache_commons_el          apache-commons-el
-%define apache_commons_fileupload  apache-commons-fileupload
-%define apache_commons_validator   apache-commons-validator
 %define java_version    1:11
 %endif
 
+%define ehcache         ( mvn(net.sf.ehcache:ehcache-core) >= 2.10.1 or ehcache-core >= 2.10.1 or ehcache >= 2.10.1)
+%define apache_commons_digester    (apache-commons-digester or jakarta-commons-digester)
+%define apache_commons_discovery   (apache-commons-discovery or jakarta-commons-discovery)
+%define apache_commons_el          (apache-commons-el or jakarta-commons-el)
```

Comment:
This can be simplified as we have on SUSE now also apache-commons-el .
I tried it and it works. Please remove jakarta-commons-el. It is not needed anymore.

Reply:
done.